### PR TITLE
libsdl2 -> 2.24.0

### PR DESCRIPTION
**Pull request by satmandu**
BUILDS properly:
- [x] `x86_64`
- [x] `i686`
- [x] `armv7l`

### Run the following to get this pull request's changes locally for testing.
```
CREW_TESTING_REPO=https://github.comsatmandu/chromebrew.git CREW_TESTING_BRANCH=libsdl2-2.24.0 CREW_TESTING=1 crew update
```

### CODE REVIEW

File: packages/libsdl2.rb [hunk 1]
```diff
@@ -3,24 +3,24 @@
 class Libsdl2 < Package
   description 'Simple DirectMedia Layer is a cross-platform development library designed to provide low level access to audio, keyboard, mouse, joystick, and graphics hardware via OpenGL and Direct3D.'
   homepage 'http://www.libsdl.org'
-  @_ver = '2.0.22'
+  @_ver = '2.24.0'
   version @_ver
-  license 'ZLIB'
   compatibility 'all'
+  license 'ZLIB'
```
Comment (uberhacker) [inline packages/libsdl2.rb:9]: Why was this line moved?

**Review by uberhacker — DISMISSED** [review]
Return license line to the original location to be consistent with other packages.

File: packages/libsdl2.rb (hunk 1 above)
Comment (satmandu) [reply packages/libsdl2.rb:9]: Rubocop...

Comment (satmandu) [reply packages/libsdl2.rb:8]: I changed it back.

**Review by supechicken — APPROVED** [review]

**Comment by satmandu** [comment]
> Return license line to the original location to be consistent with other packages.

Ideally we can have a CI app to set the right order for these?

@Zopolis4 any ideas?

**Comment by Zopolis4** [comment]
> > Return license line to the original location to be consistent with other packages.
> 
> Ideally we can have a CI app to set the right order for these?
> 
> @Zopolis4 any ideas?

I'll have a look, I can't think of any particularly neat solution right now though.
